vnext/Scripts/Microsoft.ReactNative.ProjectReunion.nuspec

```
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
-    <id>Microsoft.ReactNative.ProjectReunion</id>
+    <id>Microsoft.ReactNative.WindowsAppSDK</id>
```

The other nuspecs allow id to be set as a variable passed to the nuspec during packing. See `prep-and-pack-nuget`. Having one nuget package act differently than the others could create some confusion down the line. Could we update this one to be confistent?

```suggestion
    <id>$id$</id>
```